src/devices/sonoff.ts

```diff
@@ -655,6 +655,22 @@ const definitions: Definition[] = [
             await reporting.batteryPercentageRemaining(endpoint, {min: 3600, max: 7200});
         },
     },
+    {
+        zigbeeModel: ['KF01', 'KF-01'],
+        model: 'SNZB-01',
+        vendor: 'SONOFF',
+        whiteLabel: [{vendor: 'eWeLink', model: 'KF01'}],
```

Comment:
Can be removed

Reply:
661 and 662, these two lines are better removed? Is that what you mean because I'm completely noob. I thought they were used to identify the device. 

After I change the model, as I understand it's a new model so I will need to do PR on zigbee2mqtt to update the device database?

Reply:
Only the `whiteLabel` line can be removed. For the device database, a PR should be made with the picture:

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `SNZB-01-KF.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))